app/components/DosesDescriptionList/DosesDescriptionList.tsx

```diff
@@ -50,7 +51,8 @@ export const DosesDescriptionList = ({
       dosesPercent={dhbData.firstDosesPercentage}
     >
       <dd className={styles["vaccine-count"]}>
-        {numberFormatter.format(dhbData.firstDosesTo90Percent)} left
+        {numberFormatter.format(dhbData.firstDosesTo90Percent)} left{" "}
+        {!dhbData.hasMetFirstDoseTarget && `(${dhbData.firstDosesChange})`}
```

Comment:
IMO this should probably be formatted?

```suggestion
        {!dhbData.hasMetFirstDoseTarget && `(${numberFormatter.format(dhbData.firstDosesChange)})`}
```

Reply:
Removed from UI for now